docs/macbuild.rst

```diff
@@ -14,22 +14,25 @@ distribution to other systems.
    ``Allow apps downloaded from Anywhere``. This is necessary because
    GateKeeper will try to prevent this installation. You can return this
    setting to its previous state following the installation.
-2. Install the Python 2.7 Installer from the `Python download site`_
+2. Install the Python 2.7+ Installer from the `Python download site`_
    (double click the dmg and then double click the installer). Use the
    Mac OS X 64-bit/32-bit x86-64/i386 Installer for Mac OS X 10.6 and
    later unless you have a good reason to do otherwise.
-3. Install setuptools (if you haven’t already)
+3. Install setuptools (if you haven’t already). For Python 2 >=2.7.9 or Python 3 >=3.4,
+   pip and setuptools are already installed. Update the tools by running
+   ``pip install -U pip setuptools`` on Linux or OS
+   or python -m pip install -U pip setuptools for Windows.
 
-   -  Download `ez\_setup.py`_
+   -  Download `get-pip.py`_
    -  Navigate to the directory where you downloaded it and run
-      ``sudo python ez_setup.py``)
+      ``sudo python get-pip.py`` for Linux or OS or ``python get-pip.py`` for Windows
+      It will install or upgrade pip and setuptools and wheel if not installed
```

Comment:
I feel like setuptools should be everywhere now so we can probably remove this all of step 3.